test/js/visualization/scatter-plot.test.js

```diff
@@ -67,6 +136,24 @@ describe('getPlotlyTraces handles expression graphs', () => {
     expect(trace.marker.cmin).toEqual(0)
     expect(trace.marker.cmax).toEqual(1)
   })
+
+  it('reverses the colors of non-Reds color scales', async () => {
+    const plotData = _cloneDeep(BASIC_PLOT_DATA)
+    plotData.data.expression = [0, 0, 0, 0, 0, 0, 0, 0]
+    plotData.genes = ['foo']
+
+    let traces = getPlotlyTraces(plotData)
+    // check that it doesn't reverse the default (Reds)
+    expect(traces[0].marker.reversescale).toEqual(false)
+
+    plotData.scatterColor = 'Reds'
+    traces = getPlotlyTraces(plotData)
+    expect(traces[0].marker.reversescale).toEqual(false)
+
+    plotData.scatterColor = 'Greens'
+    traces = getPlotlyTraces(plotData)
+    expect(traces[0].marker.reversescale).toEqual(true)
```

Comment:
I understood this because I thought hard and had context, but otherwise I'd think this first block is redundant.

Please add a couple more comments for context -- perhaps:

```suggestion
    // Verify it doesn't reverse the default (Reds) when it's explicitly specified
    plotData.scatterColor = 'Reds'
    traces = getPlotlyTraces(plotData)
    expect(traces[0].marker.reversescale).toEqual(false)

    // Verify it reverses non-default color scales
    plotData.scatterColor = 'Greens'
    traces = getPlotlyTraces(plotData)
    expect(traces[0].marker.reversescale).toEqual(true)
```

Reply:
Agree that it could use more comments.  I think the important thing is to note that we're not reversing Reds.  And it's not because it is/isn't default, it's just that it's the only color scale that happens to be good for our use un-reversed.  I'll add some clarification comments